plugins/chain_interface/include/eosio/chain/plugin_interface.hpp

```diff
@@ -16,9 +16,9 @@ namespace eosio::chain::plugin_interface {
    namespace channels {
       using pre_accepted_block     = channel_decl<struct pre_accepted_block_tag,    signed_block_ptr>;
       using rejected_block         = channel_decl<struct rejected_block_tag,        signed_block_ptr>;
-      using accepted_block_header  = channel_decl<struct accepted_block_header_tag, block_state_ptr>;
-      using accepted_block         = channel_decl<struct accepted_block_tag,        block_state_ptr>;
-      using irreversible_block     = channel_decl<struct irreversible_block_tag,    block_state_ptr>;
+      using accepted_block_header  = channel_decl<struct accepted_block_header_tag, block_state_legacy_ptr>;
+      using accepted_block         = channel_decl<struct accepted_block_tag,        block_state_legacy_ptr>;
+      using irreversible_block     = channel_decl<struct irreversible_block_tag,    block_state_legacy_ptr>;
```

Comment:
Will require third-party plugins to have to be modified. They will need to be modified anyway to support instant finality. We should put some thought into what these signals should signal. Maybe `signed_block_ptr, block_id` instead.